magento/magento2
# [BUGFIX] Also set the Quantity for the product for correct tier pricing

### Description (*)

When you retrieve a cart through GraphQl after a quantity update for a product with tier pricing the single unit price is returned.

### Manual testing scenarios (*)

1. Add a product with a tier price to the cart through GraphQl
2. Change the Quantity
3. Retrieve the cart through the cart call
4. expected result: correct tier price has been applied
5. actual result: price for quantity one is returned.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29655: [BUGFIX] Also set the Quantity for the product for correct tier pricing
